### PR TITLE
Close#7507

### PR DESCRIPTION
**Pull request by GamerLab**
I have go through all the functions in Backend Api , all the methods conforms to the Array API Standard.
Although some of things can be fixed for those functions like :
1: For MxNet Any() not standard way for custom exception handling using ivy.utils.exceptions.IvyIndexError to raise an exception when encountering a RuntimeError. can cause a problem .

2: For numpy Any() no custom exception handling using ivy.utils.exceptions.IvyIndexError to raise an exception for RuntimeError. can cause a problem .

3: For paddle Any() no custom exception handling using ivy.utils.exceptions.IvyIndexError to raise an exception for RuntimeError. can cause a problem .

4: For Torch Any() no custom exception handling using ivy.utils.exceptions.IvyIndexError to raise an exception when encountering a RuntimeError. can cause a problem , also some improvements can be done in code optimization in any_torch() function, if the axis parameter is not provided (i.e., axis=None), it performs the logical OR reduction over all dimensions by calling torch.any(x, keepdim=keepdims, out=out). The function avoids using list(range(x.ndim)) to generate the default axis, making it slightly more efficient.also unique and sorted axes from the provided sequence using axis = sorted(axis) can be used .

5 : Updated the DocString for Any() in Ivy Api , try to add more examples to define all cases in Function Arguments .

### CODE REVIEW

**Comment by GamerLab** [comment]
@sagewhocodes  can you please have a look on this pull request . I have updated the doc string for Any () function .

also i want to work on more tasks i have found to be done with this Any() function for back-end function. 
can i create a new issues for these task ? 

these 4 task can be a sub task for that issue 
1: For MxNet Any() not standard way for custom exception handling using ivy.utils.exceptions.IvyIndexError to raise an exception when encountering a RuntimeError. can cause a problem .

2: For numpy Any() no custom exception handling using ivy.utils.exceptions.IvyIndexError to raise an exception for RuntimeError. can cause a problem .

3: For paddle Any() no custom exception handling using ivy.utils.exceptions.IvyIndexError to raise an exception for RuntimeError. can cause a problem .

4: For Torch Any() no custom exception handling using ivy.utils.exceptions.IvyIndexError to raise an exception when encountering a RuntimeError. can cause a problem , also some improvements can be done in code optimization in any_torch() function, if the axis parameter is not provided (i.e., axis=None), it performs the logical OR reduction over all dimensions by calling torch.any(x, keepdim=keepdims, out=out). The function avoids using list(range(x.ndim)) to generate the default axis, making it slightly more efficient.also unique and sorted axes from the provided sequence using axis = sorted(axis) can be used .

File: ivy/functional/ivy/utility.py [hunk 1]
```diff
@@ -5,11 +5,11 @@
 import ivy
 from ivy.func_wrapper import (
     handle_array_function,
+    handle_backend_invalid,
```
Comment (sagewhocodes) [inline ivy/functional/ivy/utility.py:8]: The reordering doesn't seem to be of much utility, please revert this.

File: ivy/functional/ivy/utility.py [hunk 2]
```diff
@@ -47,11 +47,11 @@ def all(
     ----------
     x
         input array.
-    axis
+    axis.
```
Comment (sagewhocodes) [inline ivy/functional/ivy/utility.py:50]: Please revert this.

File: ivy/functional/ivy/utility.py [hunk 3]
```diff
@@ -47,11 +47,11 @@ def all(
...
         axis or axes along which to perform a logical AND reduction. By default, a
         logical AND reduction must be performed over the entire array. If a tuple of
         integers, logical AND reductions must be performed over multiple axes. A valid
-        ``axis`` must be an integer on the interval ``[-N, N)``, where ``N`` is the rank
+        ``axis`` must be an integer on the interval ``[-N, N]``, where ``N`` is the rank
```
Comment (sagewhocodes) [inline ivy/functional/ivy/utility.py:54]: This correction isn't valid because by default the axes argument of numpy only allows out of order indices in the negative sense, so  `np.all(x,axes=-4)` is valid but `np.all(x,axes=4)`  is not valid.
assuming x has a shape of `(d1,d2,d3,d4)` 

Please revert this correction

File: ivy/functional/ivy/utility.py [hunk 4]
```diff
@@ -83,7 +83,7 @@ def all(
 
     Both the description and the type hints above assumes an array input for simplicit
     y,but this function is *nestable*, and therefore also accepts :class:`ivy.Container`
-    instances in place of any of the arguments.
+    instances in place of the arguments.
```
Comment (sagewhocodes) [inline ivy/functional/ivy/utility.py:86]: Please revert this change as it seems to make the documentation ambigioud

File: ivy/functional/ivy/utility.py [hunk 5]
```diff
@@ -147,92 +147,113 @@ def any(
...
-
-    >>> x = ivy.Container(a=ivy.array([0, 1, 2]), b=ivy.array([3, 4, 5]))
-    >>> y = ivy.any(x)
-    >>> print(y)
-    {
-        a: ivy.array(True),
-        b: ivy.array(True)
-    }
+    Example 1:
+    >>> import ivy
```
Comment (sagewhocodes) [inline ivy/functional/ivy/utility.py:211]: The former way of enlisting examples has been consistent across the entirety of ivy documentation, if you think this can be improved, please file a new issue. For now please revert these changes

File: ivy/functional/ivy/utility.py [hunk 6]
```diff
@@ -147,92 +147,113 @@ def any(
...
-        input array.
-    axis
-        axis or axes along which to perform a logical OR reduction. By default, a
-        logical OR reduction must be performed over the entire array. If a tuple of
-        integers, logical OR reductions must be performed over multiple axes. A valid
-        ``axis`` must be an integer on the interval ``[-N, N)``, where ``N`` is the rank
-        (number of dimensions) of ``x``. If an ``axis`` is specified as a negative
-        integer, the function must determine the axis along which to perform a reduction
-        by counting backward from the last dimension (where ``-1`` refers to the last
-        dimension). If provided an invalid ``axis``, the function must raise an
```
Comment (sagewhocodes) [inline ivy/functional/ivy/utility.py:172]: Type hints are not required in the documentation, please revert this.

**Review by sagewhocodes — CHANGES_REQUESTED** [review]

**Comment by sagewhocodes** [comment]
@GamerLab backend functions in ivy are supposed to be 100% functional logic that's defined in the framework itself ( and not ivy ) .
I would recommend you to go through Ivy's [Design](https://unify.ai/docs/ivy/overview/design.html) and [Deep Dive](https://unify.ai/docs/ivy/overview/deep_dive.html) in detail.

I appreciate you putting in the work, but since this PR doesn't actually add anything new, would you please close it? 

Looking forward to your upcoming contributions 😄
